src/interpreter_test.rs

```diff
@@ -1,6 +1,7 @@
 use parser;
-use interpreter::Interpreter;
-use interpreter::StatementResult;
+use runtime::Interpreter;
+use runtime::StatementResult;
+use ast_walk_interpreter::*;
```

Comment:
Since we need only the `AstWalkInterpreter` struct itself, I'd prefer only importing that.